pallets/ajuna-awesome-avatars/src/types.rs

```diff
@@ -18,9 +18,6 @@ use codec::{Decode, Encode, MaxEncodedLen};
 use frame_support::pallet_prelude::*;
 use scale_info::TypeInfo;
 
-pub const DNA_VERSION: u8 = 0;
```

Comment:
Let's ignore versioning for now.
If we have to update the DNA codec -> we'll add `Avatar.dna_version` field with migration starting with 0.

Reply:
Removed.